Lib: UpdateHub: Missing new timeout API conversion

**Describe the bug**
Missing new timeout api conversion.

**To Reproduce**
west build -b frdm_k64f samples/net/updatehub

**Expected behavior**
Build normally

**Impact**
Don't build

**Screenshots or console output**
```
obj   -c /home/gfbudke/zephyros/mainline/zephyr/lib/updatehub/updatehub.c
In file included from ../include/kernel_includes.h:41,
                 from ../include/kernel.h:17,
                 from ../include/zephyr.h:18,
                 from /home/gfbudke/zephyros/mainline/zephyr/lib/updatehub/updatehub.c:11:
/home/gfbudke/zephyros/mainline/zephyr/lib/updatehub/updatehub.c: In function 'wait_fds':
../include/sys_clock.h:85:28: error: incompatible type for argument 3 of 'poll'
   85 | #define Z_TIMEOUT_TICKS(t) ((k_timeout_t) { .ticks = (t) })
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                            |
      |                            k_timeout_t {aka struct <anonymous>}
../include/sys_clock.h:87:25: note: in expansion of macro 'Z_TIMEOUT_TICKS'
   87 | #define Z_TIMEOUT_MS(t) Z_TIMEOUT_TICKS(k_ms_to_ticks_ceil32(MAX(t, 0)))
      |                         ^~~~~~~~~~~~~~~
../include/kernel.h:1585:24: note: in expansion of macro 'Z_TIMEOUT_MS'
 1585 | #define K_MSEC(ms)     Z_TIMEOUT_MS(ms)
      |                        ^~~~~~~~~~~~
../include/kernel.h:1597:24: note: in expansion of macro 'K_MSEC'
 1597 | #define K_SECONDS(s)   K_MSEC((s) * MSEC_PER_SEC)
      |                        ^~~~~~
/home/gfbudke/zephyros/mainline/zephyr/lib/updatehub/updatehub.c:35:25: note: in expansion of macro 'K_SECONDS'
   35 | #define NETWORK_TIMEOUT K_SECONDS(2)
      |                         ^~~~~~~~~
/home/gfbudke/zephyros/mainline/zephyr/lib/updatehub/updatehub.c:98:30: note: in expansion of macro 'NETWORK_TIMEOUT'
   98 |  if (poll(ctx.fds, ctx.nfds, NETWORK_TIMEOUT) < 0) {
      |                              ^~~~~~~~~~~~~~~
In file included from /home/gfbudke/zephyros/mainline/zephyr/lib/updatehub/updatehub.c:14:
../include/net/socket.h:651:64: note: expected 'int' but argument is of type 'k_timeout_t' {aka 'struct <anonymous>'}
  651 | static inline int poll(struct zsock_pollfd *fds, int nfds, int timeout)
      |                                                            ~~~~^~~~~~~
[129/189] Building C object zephyr/subsys/net/ip/CMakeFiles/subsys__net__ip.dir/net_if.c.obj
ninja: build stopped: subcommand failed.
FATAL ERROR: command exited with status 1: /usr/bin/cmake --build 

```
**Environment (please complete the following information):**
 - OS: Linux
 - Toolchain Zephyr SDK 0.11.2
 - Commit SHA ed2d263

--- a/lib/updatehub/updatehub.c
+++ b/lib/updatehub/updatehub.c
@@ -32,7 +32,7 @@ LOG_MODULE_REGISTER(updatehub);
 #include <net/tls_credentials.h>
 #endif
 
-#define NETWORK_TIMEOUT K_SECONDS(2)
+#define NETWORK_TIMEOUT (2 * MSEC_PER_SEC)
 #define UPDATEHUB_POLL_INTERVAL K_MINUTES(CONFIG_UPDATEHUB_POLL_INTERVAL)
 #define MAX_PATH_SIZE 255
 /* MAX_PAYLOAD_SIZE must reflect size COAP_BLOCK_x option */

